tuned/verifylog.py

```diff
@@ -0,0 +1,108 @@
+
+import json
+
+class VerifyLog:
+
+	obj = None
+
+	def __init__(self, log={}):
+		self.vLog = log
+		return
+
+	@classmethod
+	def get_obj(self, log={}):
+		"""
+		Create and return a singleton object.
+		"""
+		if (not self.obj):
+			self.obj = VerifyLog(log)
+		return self.obj
+
+	def add_log(self, iname, log={}):
+		if (iname not in self.vLog.keys()):
+			self.vLog[iname] = []
+		if (log):
+			self.vLog[iname].append(log)
+
+		return
+
+	def get_log(self):
+		return self.vLog
+
+	def put_log(self):
+		vlog = self.vLog
+		self.vLog = {}
+		return vlog
+
+	def _display_device(self, log):
+		ret = True
+		for i in range(len(log)):
```

Comment:
I would rewrite these functions more pythonic/readable. Var i is used just for index, so it can be "for l in log" or smth.